5 - RPG/Code/ClassLibrary1/CharacterBuilder.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Characters
+{
+    public class CharacterBuilder
+    {
+
+
+            public Character BuildCharacter(int head, FitnessLevel fit, int heightInInches, List<AbilityPoints> distributedPoints)
+            {
+
+            int expectedAbilityPointTotal = 60;
+
+                if (head < 0 || head > 100)
+                {
+                    throw new Exception("Unable to create character headShape should be between 0 and 100.\n");
+                }
+
+                Dictionary<string, AbilityPoints> validatedAbilities = new Dictionary<string, AbilityPoints>();
+
+                string errormessages = "";
+                var total = 0;
+                foreach (AbilityPoints ability in distributedPoints)
+                {
+                    total += ability.Value;
```

Comment:
Since you already have all your Abilities in a list you can use linq to get the total like:
```
var total = distributedPoints.Sum(x => x.Value)
```